### ISSUE TITLE
Modality is broken

### ISSUE THREAD

**Issue by webmaster128** [kurman-vladimir/qml-QFileDialog-wrapper#2]
This should not be possible

![broken_modality](https://cloud.githubusercontent.com/assets/2603011/12508231/21409c0e-c0fb-11e5-8953-30dc40687ba8.png)

**Comment by kurman-vladimir** [comment]
Is this solution appropriate for you?

**Comment by webmaster128** [comment]
Did you test it? The issue is the same as before. As far as I recall, modality only works when QFileDialog has a parent set.

**Comment by kurman-vladimir** [comment]
Hm... It works fine on my system (CentOS 7/Gnome 3). As I see you are using KDE. I will try it in your environment.

**Comment by webmaster128** [comment]
I'm on Ubuntu 15.10 (Unity)

**Comment by kurman-vladimir** [comment]
Year. You are right. This solution doesn't work on Ubuntu 15.10. That's a pity.

**Comment by webmaster128** [comment]
Tested it on Windows Server. There it works fine.

**Comment by kurman-vladimir** [comment]
Yes. This works on Windows and Linux with Gnome environment. I will try to workaround KDE issue.
